packages/react/src/exampleTypes/ExpandableTagsQuery/index.js

```diff
@@ -166,10 +109,15 @@ let TagsWrapper = observer(
     splitCommas = true,
     maxTags = 1000,
     hasPopover,
-    generationsCollapsed: generationsCollapse,
     enableKeywordGenerations,
+    generationsCollapsed: generationsCollapse,
     ...props
   }) => {
+    //Handle Outside Clicks for Keyword Generations display
+    let ref = React.useRef()
+    let generationsCollapsed = React.useState(true)
```

Comment:
We already have `generationsCollapsed` passed as a prop to this component, why do we need to define another one? 

If we absolutely need to define another one you need to check is there is popup open in the outside click handler like we do here

https://github.com/smartprocure/contexture/blob/349793ca2b3459ae25c5185870bf90a8904c399d/packages/react/src/exampleTypes/ExpandableTagsQuery/index.js#L60

Reply:
I updated the code appropriately